apps/site/pages/en/blog/migrations/v22-to-v24.mdx

```diff
@@ -0,0 +1,238 @@
+---
+date: '2025-10-21T12:03:00.000Z'
```

Comment:
Probably ought to update the date so it is at least the release date of the referenced Node.js 24.11.0 release or later.